src/types/options/index.ts

```diff
@@ -52,6 +52,13 @@ export interface OptionsRequest<Source extends OptionsSource = OptionsSource> ex
 
   // this appears in the block_suggestions schema, but we're not sure when its present or what its type would be
   app_unfurl?: any;
+
+  // exists for enterprise installs
+  is_enterprise_install?: boolean;
```

Comment:
Haven't confirmed if the payload is actually getting these two keys yet in enterprise install (and that team is null)